zipline/protocol.py

```diff
@@ -506,6 +545,26 @@ def has_key(self, name):
         """
         return name in self
 
+    def get_default(self, name):
```

Comment:
So this is a `defaultdict` like behavior? The `get_default` name feels a little disjointed, since it doesn't always return the default, maybe something like `get_or_default` or just `get`?